h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -116,15 +152,27 @@ export default function CreateGroupForm() {
         />
         <TextField
           type="textarea"
+          value={description}
+          onChangeValue={setDescription}
           limit={250}
           label="Description"
           testid="description"
           classes="h-24"
         />
 
-        <div className="flex">
+        <div className="flex items-center">
+          {errorMessage ? (
+            <div
+              className="text-red-error font-bold"
+              data-testid="error-message"
+            >
+              {errorMessage}
+            </div>
+          ) : null}
```

Comment:
The convention for conditional rendering with no "else" block is to use `{condition && jsx_expression}`.

Reply:
Done